app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PagesController < ApplicationController
+  include HighVoltage::StaticPage
+
+  before_action :authorize_page, only: [:update, :show, :edit]
+
+  private
+
+  def authorize_page
+    authorize :page
```

Comment:
Please check this.  For headless policies (https://github.com/elabs/pundit#headless-policies) the authorization action looks like this:
```
authorize :page, show?
```
(using `show` action as an example)